### PR TITLE
About and go PRO Page+ Hover

### PR DESCRIPTION
**Pull request by giannamoth**
Prepared basic About and go PRO pages. They are connected with the Href, pls check if it is all good :)
Furthermore I've added some hovering color changes on the buttons and icons with links.

ACHTUNG!!!!! the go PRO page is still a prototype. My goal is to have it presentable for the mid-term review

### CODE REVIEW

**Comment by giannamoth** [comment]
edit: the go PRO page is MVP ready. Please check :)

**Comment by vagechirkov** [comment]
New pro icons are amazing!

<details>

image


</details>

File: node_app/client/about.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </div>
+    </header>
+
+    <div>
+        <img id="background" src="content/animatedWaves.gif" alt="">
+    </div>
+
+    <div class="canvas">
+
+        <div id="about">
```
Comment (vagechirkov) [inline node_app/client/about.html]: `about` id is duplicated in line 23

File: node_app/client/index.html [hunk 2]
```diff
@@ -201,7 +200,7 @@ <h4 class="interTitle">Where you can use it</h4>
         </div>
 
         <form>
-            <div id="formGenerate">
+            <div id="formGenerate2">
```
Comment (vagechirkov) [inline node_app/client/index.html]: `formGenerate` is duplicated; this might cause an error

**Comment by vagechirkov** [comment]
I have updated [deployment on Heroku](http://noise-enricher.herokuapp.com/index.html), but for now, it always opens `index.html` when the form is submitted. I think we need to discuss what should happen when the form is submitted.

File: node_app/client/PRO.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    <title>The Noise Enricher-PRO</title>
+</head>
+
+<body>
+
+    <header>
+        <h4 id="logo"><a href="index.html">The Noise Enricher</a></h4>
+        <div id="menu">
+            <h4 id="About"><a href="about.html">About</a></h4>
+            <h4 id="logIn"><a href="logIn.html">Log in</a></h4>
```
Comment (vagechirkov) [inline node_app/client/PRO.html:23]: this file does not exist

**Review by vagechirkov — COMMENTED** [review]
The hover effect works excellent!  I think this PR is ready to merge after fixing some minor issues mentioned above.

Overall, I think we might want to reduce duplications of the content using [ejs templates ](https://ejs.co/), for instance.

File: node_app/client/about.html (hunk 1 above)
Comment (giannamoth) [reply node_app/client/about.html]: fixed :)

File: node_app/client/index.html (hunk 2 above)
Comment (giannamoth) [reply node_app/client/index.html]: fixed :)

File: node_app/client/PRO.html (hunk 3 above)
Comment (giannamoth) [reply node_app/client/PRO.html:23]: do you mean the logIn.html? if yes, I still have to  make it

**Comment by giannamoth** [comment]
sooo, IVe fixed all the issues mentioned above. And yess, I'm up to look for implementations that are going to remove duplications and so on.

Let's figure this out after the mid-term presentation 😎

**Comment by vagechirkov** [comment]
> sooo, IVe fixed all the issues mentioned above.

@giannamoth I don't see any commits to this PR ([giannaDevelop branch](https://github.com/giannamoth/wt21-the-noise-enricher/commits/giannaDevelop)). Did you push changes?

**Comment by giannamoth** [comment]
god I didn'T xD I did now

File: node_app/client/about.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                similique
+                maxime
+                optio suscipit sapiente? Eius ea eum, saepe, accusantium recusandae dolore optio repudiandae ad est
+                aperiam
+                quas?
+            </p>
+        </div>
+    </div>
+    <footer>
+        <script src="index.js"></script>
```
Comment (vagechirkov) [inline node_app/client/about.html:62]:
```suggestion
        <script src="js/index.js"></script>
```

**Comment by vagechirkov** [comment]
LGTM!

**Review by vagechirkov — APPROVED** [review]

**Pull request merged**